__test__/Ship.test.js

```diff
@@ -0,0 +1,63 @@
+const Ship = require('../src/Ship.js');
+
+describe('Ship', () => {
+    describe('with ports and an itinerary', () => {
+        let ship;
+        let dover;
+        let calais;
+        let itinerary;
+
+        beforeEach(() => {
+        dover = {
+            addShip: jest.fn(),
+            removeShip: jest.fn(),
+            name: 'Dover',
+            ships: []
+        };
+        calais = {
+            addShip: jest.fn(),
+            removeShip: jest.fn(),
+            name: 'Calais',
+            ships: []
+        };
+        itinerary = {
+            ports: [dover, calais]
+        };
+        ship = new Ship(itinerary);
+    });
```

Comment:
All this should be indented by 4 spaces so it fits properly inside the `beforeEach` function